recipes/onqtam/doctest/package.txt

```diff
@@ -0,0 +1,2 @@
+onqtam/doctest@master
```

Comment:
Is there a way to use a version tag instead of always pulling from master?

Reply:
Hmmm, yes the repo does seem to have tags for each of its releases, so I guess those could be used instead.

Is it generally preferred for recipes not to pull from master?

Reply:
> Is it generally preferred for recipes not to pull from master?

Yes, as it helps keep the recipes reproducible.

Reply:
Ahha, I see. Yes that makes sense.

Does that also imply then, that if two different users both wanted reproducible setups,  but using two different versions of a library, then we might want two different recipes for that same library (eg. with the version number in the recipe name perhaps) in cget?

Apologies for all the questions, but I'm genuinely interested in finding out what the ideal cget workflow is.

Reply:
Cget doesn't really support installing two different versions in the same cget directory(as two packages are not allowed to install the same file).

You can also add multiple versions of recipe under the same directory, such as `onqtam/doctest/1.0/package.txt` and `onqtam/doctest/2.0/package.txt` and then you can refer to it as `onqtam/doctest@1.0` or `onqtam/doctest@2.0`. Both packages are given the same name as `onqtam/doctest`. Currently, there is no SAT solving, just first-come-first-serve(ie cget just checks if the package is already installed, and then installs). So to fix a correct version, the user should install it first or use a set of recipes that are know to work together.

Of course, this recipe is only useful for picking a usable version. So I dont see the need to add multiple recipes as the user can already do `onqtam/doctest@1.0` to pick a certain tag and install.